testutil/cert.go

```diff
@@ -0,0 +1,84 @@
+package testutil
+
+import (
+	"crypto/ecdsa"
+	"crypto/elliptic"
+	"crypto/rand"
+	"crypto/tls"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/asn1"
+	"encoding/pem"
+	"math/big"
+	"time"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/ovrclk/akash/x/cert/types"
+)
+
+var AuthVersionOID = asn1.ObjectIdentifier{2, 23, 133, 2, 6}
```

Comment:
It'd be nice to have a comment explaining what these numbers mean

Reply:
thats defined here https://portal.etsi.org/pnns/asn1oids